CODESTYLE.md

```diff
@@ -4,6 +4,19 @@
 - Start header files with `#pragma once`.
 - Use camelcase for naming.
 - The `{` is placed on the next line after a function statement.
+  
+  Example:
```

Comment:
Why example of if statement when the item is about a function statement

Reply:
It is meant to be an example of the use of the brackets in any function statement. If statement being an easy example of how to use it with the if and the else statements.

Reply:
if statement isn't a function statement.